app/components/forms.tsx

```diff
@@ -5,6 +5,14 @@ import { Label } from "@/components/ui/label";
 
 export type ListOfErrors = (string | null | undefined)[] | null | undefined;
 
+interface FieldProps {
```

Comment:
Utiliser plutôt type

Reply:
j'ai l'erreur : "Use an `interface` instead of a `type`."

Reply:
c'est fait